examples/lightgcn_example.py

```diff
@@ -33,6 +33,18 @@
     rating_threshold=0.5,
 )
 
+lightgcn1 = cornac.models.LightGCN1(
+    name="LightGCN-old",
+    seed=123,
+    num_epochs=2000,
+    num_layers=3,
+    early_stopping={"min_delta": 1e-4, "patience": 3},
```

Comment:
I guess we can make the hyper-parameters similar to the example of NGCF?

Reply:
Actually it already closely follows NGCF, just that 'num_layers' instead of 'layer_sizes' because we dont need to specify feature size for each layer.

Reply:
Based on [this](https://github.com/PreferredAI/cornac/blob/master/examples/ngcf_example.py), there are a few things we can align:
```python
num_epochs=1000
early_stopping={"min_delta": 1e-4, "patience": 50}
train_batch_size=1024
lambda_reg=1e-5
```

Reply:
Oh yes, you are right. But for lambda_reg I think in lightGCN paper they mentioned 1e-4 is optimal. Should we keep that, or follow NGCF?

Reply:
We can keep 1e-4 as in the paper